packages/autoprofile/lib/samplers/async_sampler.js

```diff
@@ -7,7 +7,7 @@
 
 'use strict';
 
-const fs = require('fs');
+const { uninstrumentedFs: fs } = require('@instana/core');
```

Comment:
TBD: Add a utility npm package `@instana/utils` to serve libs such uninstrumentedFs etc.

Reply:
I don't have a strong opinion, I think. All the packages that would use that also either rely on `core` directly or indirectly, or are always installed together with `core`. (`autoprofiler` is never used directly, but only as a dependency of `collector`, which in turn requires `core`). But I agree, adding `core` just for this feels weird.

Let's discuss pros and cons in person at some point.

Reply:
So do we want to keep this as it is and discuss+create a ticket to add a utility package?

Reply:
Yes, good idea, I think that would be best. We can merge this as-is and refactor later. This would also help keep the scope of this PR smaller. Or are you strongly leaning towards doing this in the scope of this PR?

Reply:
👍